### ISSUE TITLE
poped - evaluate_power error in example 11

### ISSUE THREAD

**Issue by TheoPapath** [nlmixr2/babelmixr2#135]
evaluate_power gives an error

![image](https://github.com/user-attachments/assets/c7be89c0-0898-472a-bd97-de7780e89fcd)

This happens even though parameter number 5 is not fixed - likely a poped error and nothing can be done
![image](https://github.com/user-attachments/assets/03dae173-4150-4646-8147-26927cbc122d)

Also in this example, the pediatric FIM does not match 1:1 with the poped examples. Not sure why that is.

**Comment by mattfidler** [comment]
Hi @TheoPapath 

The reason the pediatric `FIM` does not match is because you mis-translated the model using:

```r
    KA<-tKa*exp(eta.ka)* (pedCL^(isPediatric)) # add covariate for pediatrics
```

Once you correct to:

```r
    CL<-tCl*exp(eta.cl)* (pedCL^(isPediatric)) # add covariate for pediatrics
```

The `FIM` matches.   I have updated the example in your branch (you will need to pull the changes).

Also in the `babelmixr2` translation the model does not have a parameter `5`, so this is the reason this does not exist.  If you update this to index `4` it works just fine.

However, this is really not that user-friendly, so I have added the function:

```r
babelBpopIdx(babel.db.all, "pedCL")
```

Which will query the `babel.db.all` for the "pedCL" parameter and give the correct index.  This is in #136
